azavea/fb-gender-survey-dashboard
# Add base visualization section with a data indexer

## Overview

An initial pass at a format to provide question/response pairs based on
visualization type for the selected question. It is anticipated that
additional shaping will be required for certain chart types, but this lays the
pathway with a focus on accounting for the "stack" type which aggregates
a number of variables/response from a single question. Other chart types
will need similar custom shaping.

Some additional changes were made based on updates to the data file received throughout the course of development.

Connects #8 

## Testing Instructions

 * Select a "stack" type question ("do you strongly agree...") and a non-stack type question
 * Progress to the final "visualization" page and inspect the processed data that is output to the screen (it may be useful to copy to a json viewer like http://jsonviewer.stack.hu/)
 * Check to see that the stack type group the various variables associated with the question together, and that other types behave differently.
 * Select multiple geographies and ensure that a each question/response is also sliced by geography
